### PR TITLE
SP23 sprint6 PR#3

### PR DESCRIPTION
**Pull request by simranmakandar98**
- mansi's code changes (apply filter, anchor to top of page)
- django bumped up to 4.1.9 (dependabot suggestion)
- examples for cypress.env
- READme update
- cypress progress tracker update

### CODE REVIEW

File: projects/templates/projects/allProjects.html [hunk 1]
```diff
@@ -227,7 +227,7 @@
                     false
                 );
             </script>
-            <input type="button" id="btnApply" value="Apply Filters" class="btn btn-secondary" data-cy="applyfilters"/>
+            <input type="button" id="btnApply" value="Apply Filters" class="btn btn-secondary" style="background-color: crimson;" style="background-color: crimson;" data-cy="applyfilters"/>
```
Comment (SachinPawaskarUNO) [inline projects/templates/projects/allProjects.html:230]: style="background-color: crimson;"  is there twice.

File: projects/templates/reports/projectsprivatetableview.html [hunk 2]
```diff
@@ -232,7 +232,7 @@
                     false
                 );
             </script>
-             <input type="button" id="btnApply" value="Apply Filters" class="btn btn-secondary" data-cy="applyfilters"/>
+             <input type="button" id="btnApply" value="Apply Filters" class="btn btn-secondary" style="background-color: crimson;" style="background-color: crimson;" data-cy="applyfilters"/>
```
Comment (SachinPawaskarUNO) [inline projects/templates/reports/projectsprivatetableview.html:235]: style="background-color: crimson;"  is there twice

File: README.md [hunk 3]
```diff
@@ -53,4 +54,8 @@ Navigate to the folder containing manage.py and run the following commands in or
 
 Test Server:
 
-Navigate to the folder containing manage.py and run the following command. python manage.py runserver navigate to http://127.0.0.1:8000/ to go to the home page.
+Navigate to the folder containing manage.py and run the following command. 
+
+    python manage.py runserver
```
Comment (SachinPawaskarUNO) [inline README.md:59]: Put code that can be run like this in 
```
python manage.py runserver 
```

File: projects/templates/reports/projectspublictableview.html [hunk 4]
```diff
@@ -215,7 +215,7 @@
                     false
                 );
             </script>
-            <input type="button" id="btnApply" value="Apply Filters" class="btn btn-secondary" data-cy="applyfilters"/>
+            <input type="button" id="btnApply" value="Apply Filters" class="btn btn-secondary" style="background-color: crimson;" style="background-color: crimson;"data-cy="applyfilters"/>
```
Comment (SachinPawaskarUNO) [inline projects/templates/reports/projectspublictableview.html:218]: style="background-color: crimson;" 
remove duplicate

**Review by SachinPawaskarUNO — CHANGES_REQUESTED** [review]
@simranmakandar98 and @MansiKamthan 
Please see review comments

**Comment by simranmakandar98** [comment]
Thank you for your review, Dr. Pawaskar @SachinPawaskarUNO 

we are making changes as suggested
@MansiKamthan

**Review by SachinPawaskarUNO — APPROVED** [review]
Looks good

**Pull request merged**